app/controllers/pg_hero/home_controller.rb

```diff
@@ -80,7 +80,9 @@ def space
       @unused_index_names = Set.new(@unused_indexes.map { |r| r[:index] })
       @show_migrations = PgHero.show_migrations
       @system_stats_enabled = @database.system_stats_enabled?
-      @index_bloat = @database.index_bloat
```

Comment:
https://github.com/ankane/pghero/blob/ad182230c1f1890662c9525b5778f70880f34601/app/controllers/pg_hero/home_controller.rb#L93

for what its worth it looks like they did the same thing here?